CONTRIBUTING.md

```diff
@@ -271,6 +272,19 @@ If you have made more than one commit, then you will need to _squash_ your commi
 
 To do this, see [Squashing Your Commits](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231).
 
+### Commit messages
+When you commit your changes, please use conventional commit messages.
+
+The commit message should be structured as follows:
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer]
+```
+Learn more at [Conventional Commits](http://conventionalcommits.org).
+
```

Comment:
I would add:

"For help writing your commit message, execute `npm run commit` from the command line and the [commitizen](https://github.com/commitizen/cz-cli) CLI tool will assist you in creating a conventional commit message."

Reply:
Thanks @dhcodes . I will do the changes.